borg/xattr.py

```diff
@@ -113,7 +113,7 @@ def split_lstring(buf):
     return result
 
 
-class BufferTooSmallError(Exception):
+class BufferTooSmallError(OSError):
```

Comment:
did you check whether this triggers secondary errors, e.g. because handler code assumes that exception object has errno attribute?